src/components/User/UserProfile.tsx

```diff
@@ -134,22 +149,19 @@ const Value = styled.div`
   border-radius: 10px;
   background: ${({ theme }) => theme.color.background.card};
   color: ${({ theme }) => theme.color.text.card};
+  font-size: 15px;
+  font-family: 'PTRootUIWebMedium', sans-serif;
```

Comment:
I've always had this question of why we're enforcing fonts throughout the project. Do we need to do that?

Reply:
@MayukhSamadder2000 Yes. In the project we are using this font family to display some important text.
Without this font family, it is not align with the design. so I have used that font-family.
![image](https://user-images.githubusercontent.com/109526304/230996709-30dba444-bc10-48d9-8381-a096ae92d2e7.png)